packages/lesswrong/components/posts/PostsAnnualReviewMarketTag.tsx

```diff
@@ -0,0 +1,81 @@
+import { AnnualReviewMarketInfo } from '../../lib/annualReviewMarkets';
+import { useSingle } from '../../lib/crud/withSingle';
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import React from 'react';
+import { useHover } from '../common/withHover';
+
+const styles = (theme: ThemeType) => ({
+  expectedWinner: {
+    color: theme.palette.review.winner,
+    border: '1px solid',
+    borderColor: theme.palette.review.winner,
+    borderRadius: '4px',
+    fontFamily: theme.typography.fontFamily,
+    width: 'fit-content',
+    display: 'inline-block',
+    padding: '5px',
+    boxSizing: 'border-box',
+  },
+  preview: {
+    maxWidth: 400,
+  },
+});
+
+const PostsAnnualReviewMarketTag = ({ post, annualReviewMarketInfo, classes }: {
+  post: PostsWithNavigation | PostsWithNavigationAndRevision | PostsList,
+  annualReviewMarketInfo?: AnnualReviewMarketInfo,
```

Comment:
I think `annualReviewMarketInfo` shouldn't be optional, since the only usage I see of `PostsAnnualReviewMarketTag` is in a context where we're checking that `annualReviewMarketInfo` exists